### PR TITLE
Format codebase using djade

### PR DESCRIPTION
**Pull request by johannaengland**
Now we can discuss if we prefer djade or djLint (#182) to format HTML files

Djade documentation: https://github.com/adamchainz/djade

### CODE REVIEW

File: src/argus_htmx/templates/htmx/incidents/_incident_row.html [hunk 1]
```diff
@@ -2,20 +2,20 @@
...
+	  {% endblock tag %}
 	  {% block start_time %}
 	  {% include "htmx/incidents/_incident_start_time.html" %}
-	  {% endblock %}
+	  {% endblock start_time %}
 	  {% block description %}
 	  {% include "htmx/incidents/_incident_description.html" %}
-	  {% endblock %}
+	  {% endblock description %}
   </li>
```
Comment (elfjes) [inline src/argus_htmx/templates/htmx/incidents/_incident_row.html:21]: I don't think incident_row.html is used anymore. We should get rid of (and also the unused ìncident_row` function in incidents/views.py)

**Review by elfjes — COMMENTED** [review]
It doesn't seem to do much, does it?

**Comment by johannaengland** [comment]
> It doesn't seem to do much, does it?

No, since it focuses on Django templates and not the underlying HTML

**Comment by johannaengland** [comment]
We decided to go for djLint for formatting instead since we cannot use both at the same time (due to endblock rules on the same line) and djade alone is not enough.

**Pull request closed without merging**